README.md

```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
   - Configure journeys
-  - Communication service: communications between journeys
-  - Simple examples of journeys such as transactions and transfer
+    - This is [an example](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/lib/services/transactions-journey-config.service.ts) of how the journey configuration might be defined in the application 
+    - By default, it will provide some value intended by the journey developers and exported by public API. Check the [`index.ts`](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/index.ts#L1), however, you can customize values during the creation of applications like [`transactions-journey-bundle.module.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/transactions/transactions-journey-bundle.module.ts#L15)
+  - Communication service or how to communicate between journeys
+
+    - There are 3 parts of the communication chain:
+      - Source journey (the one that will send some data or signal) should define and export. Check the [`make-transfer-communication.service.ts`](https://github.com/Backbase/golden-sample-app/blob/master/libs/transfer/src/lib/services/make-transfer-communication.service.ts)
+      - The destination journey (the one that will receive the data of signal) should also define the interface of the service that it expects. Check the [`communication/index.ts`](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/lib/communication/index.ts)
+      - The actual implementation of the service lives on the application level and must implement both of the available interfaces (or abstract classes) from source and destination journeys. Check the example [`journey-communication.service.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/services/journey-communication.service.ts)
+      - Do not forget, that communication service from the application level should be provided to the journeys modules in the bundle files (to avoid breaking lazy loading). Check the [`transactions-journey-bundle.module.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/transactions/transactions-journey-bundle.module.ts)
+    - The general explanation of the communication service idea and  its theoretical underlying can be found in [communicate_between_journeys](https://community.backbase.com/documentation/foundation_angular/latest/communicate_between_journeys)
```

Comment:
```suggestion
    - The general explanation of the communication service idea and  its theoretical underlying can be found in [Understand communication between journeys](https://community.backbase.com/documentation/foundation_angular/latest/communicate_between_journeys)
```

Reply:
I suppose it's better to use the page title here

Reply:
That's a good suggestion